[2022-12-25T19:52:09.802+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-10-27T15:00:00+00:00 [queued]>
[2022-12-25T19:52:09.814+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-10-27T15:00:00+00:00 [queued]>
[2022-12-25T19:52:09.815+0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:52:09.815+0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2022-12-25T19:52:09.816+0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:52:09.833+0300] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): first_task> on 2022-10-27 15:00:00+00:00
[2022-12-25T19:52:09.839+0300] {standard_task_runner.py:55} INFO - Started process 34305 to run task
[2022-12-25T19:52:09.857+0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_pivot', 'first_task', 'scheduled__2022-10-27T15:00:00+00:00', '--job-id', '741', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_total.py', '--cfg-path', '/var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T/tmppvdfpc4m']
[2022-12-25T19:52:09.861+0300] {standard_task_runner.py:83} INFO - Job 741: Subtask first_task
[2022-12-25T19:52:09.891+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:52:09.893+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:52:09.896+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:400 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:52:09.898+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_engine_encoding] has been deprecated, you should use[database/sql_engine_encoding] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:52:09.931+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/utils/sqlalchemy.py:121 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:52:09.992+0300] {task_command.py:389} INFO - Running <TaskInstance: titanic_pivot.first_task scheduled__2022-10-27T15:00:00+00:00 [running]> on host mbp-vadim.bbrouter
[2022-12-25T19:52:10.263+0300] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-10-27T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-27T15:00:00+00:00
[2022-12-25T19:52:10.265+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T
[2022-12-25T19:52:10.266+0300] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Here we start!"']
[2022-12-25T19:52:10.278+0300] {subprocess.py:86} INFO - Output:
[2022-12-25T19:52:10.282+0300] {subprocess.py:93} INFO - Here we start!
[2022-12-25T19:52:10.282+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-25T19:52:10.324+0300] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=first_task, execution_date=20221027T150000, start_date=20221225T165209, end_date=20221225T165210
[2022-12-25T19:52:10.374+0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-25T19:52:10.431+0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
